openshift/README.md

```diff
@@ -1,5 +1,19 @@
 # Openshift green field deployment of fabric8-analytics services
 
+## Install required tools
+
+Use your preferred package manager to install awk, aws-cli, psql, origin-clients, pwgen
+
+
+## Configure aws cli
```

Comment:
Do we need this? I think env. vars should take precedence over the config file.

Reply:
Hmm Looking into aws-cli documentation it should work with env vars. but it seems we are missing AWS_DEFAULT_REGION.